rollup-cli/tools/helpers/sendBot.js

```diff
@@ -90,7 +90,7 @@ async function send(UrlOperator, idTo, amount, walletJson, password, tokenId, us
                 if (codeWrongTransaction === 11) { // -sign don't matchthe Tx
                     transaction.toIdx += 1;
                 }
-                axios.post(`${UrlOperator}/offchain/send`, parsetransaction)
+                apiOperator.sendOffChainTx(parsetransaction)
```

Comment:
We should use lowerCamelCase naming convention:
`parsetransaction` to `parseTransaction`

Reply:
thanks ^^